{{ cookiecutter.package_name }}/pytest.ini

```diff
@@ -0,0 +1,27 @@
+[pytest]
+minversion = 7.0
+testpaths =
+  {{ cookiecutter.module_name }}
+  docs
+norecursedirs =
+    .tox
+    build
+    docs/_build
+    docs/generated
+    *.egg-info
+    examples
+    {{ cookiecutter.module_name }}/_dev
+    .history
+    {{ cookiecutter.module_name }}/extern
+doctest_plus = enabled
+doctest_optionflags = NORMALIZE_WHITESPACE FLOAT_CMP ELLIPSIS
+text_file_format = rst
+addopts = --doctest-rst
```

Comment:
```suggestion
addopts = --doctest-rst  -p no:unraisableexception -p no:threadexception
```

Reply:
Apparently these are things we need @nabobalis ?

Reply:
Yeah, for online downloads we normally get  these errors raised by pytest but there is not anything we can do about it as far as I understand.

Reply:
Do we need them everywhere or should we only do them in core / repos where it's a problem?

Reply:
I'm not aware of any issues it could hide. But that doesn't mean it won't. 

 I've had to add them to several repositories at this point
Mostly ones with online tests.